font-codegen/src/fields.rs

```diff
@@ -402,7 +402,7 @@ fn traversal_arm_for_field(
             quote!(Field::new(#name_str, traversal::FieldType::var_array(#typ_str, self.#name()#maybe_unwrap, #data)))
         }
         // HACK: who wouldn't want to hard-code ValueRecord handling
```

Comment:
I know it's not new but lets change // HACK to // issue link to fix the value record hacks

Reply:
#659 for tracking all codegen hacks